calcom/cal.com
# feat: Adjusted Recurring Event Booking Menu to Follow Google Calendar-like Structure

…Format

## What does this PR do?
Adjusted the menu when setting a recurring event to be more similar to GCal.
Specifically, set the default values to be yearly, monthly, weekly, daily, weekdays and custom.
Toggling custom allows selection of intervals of x days/weeks/months/years

Added an option to denote meetings should continue indefinitely-sets count behind the scenes to max value.
Otherwise, allows both selection of max meeting numbers and/or a definite end date.

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



Tested this by running development and making sure that all the updates saved correctly. We checked all references to changed variables to make sure nothing else was affected.

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
